spec/controllers/mailgun_webhooks_controller_spec.rb

```diff
@@ -93,8 +93,10 @@
             end.to change(IncomingEmail, :count).by(0).and change(Client, :count).by(0)
             expect(IntercomService).to have_received(:create_message).with(
               email_address: sender_email,
-              inform_of_handoff: false,
-              body: "Hi Alice,\n\nThis is Bob.\n\nI also attached a file."
+              body: "Hi Alice,\n\nThis is Bob.\n\nI also attached a file.",
```

Comment:
The text here says that Bob attached a file but we're not doing `has_documents: true`. Let's see if we should fix the copy here.